### DISCUSSION TITLE
add @gurgunday to fastify core team

### DISCUSSION THREAD

**Discussion by Uzlopak** [fastify/fastify#5440] [General]
@fastify/leads 

I think the effort of @gurgunday for the fastify core project is significant enough and we should add him to the core team.

**Comment by mcollina** [comment]
@Eomm @jsumners PTAL

**Reply by Eomm** [reply]
Yeah, totally agree!

**Reply by jsumners** [reply]
Agreed.

**Comment by gurgunday** [comment]
Funnily enough I was actually planning on making an official request after the v5 release, had a few more PRs in mind

Thank you for nominating me again, it's in large part thanks to you that I found my place here ♥️

**Comment by mcollina** [comment]
Done
